release-it.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  plugins: {
+    'release-it-lerna-changelog': {
+      infile: 'CHANGELOG.md',
+    },
+  },
+  git: {
+    commitMessage: 'v${version}',
+    tagName: 'v${version}',
+  },
+  github: {
+    release: true,
+    releaseName: 'v${version}',
+    tokenRef: 'GITHUB_AUTH',
+  },
+  npm: {
+    publish: false,
+  },
```

Comment:
For my learning and understanding, why do we plan to publish in GitHub's Registry and not in NPM?

Reply:
Ok now I see this PR: https://github.com/simplabs/ember2-x-codemods/pull/5/files, so the workflow is to publish on GitHub manually (and test the package?) before publishing it into NPM?

Reply:
Yeah so it runs the test workflow before publishing